addons/sale/wizard/res_config_settings_views.xml

```diff
@@ -161,6 +161,10 @@
                             <field name="module_sale_amazon" widget="upgrade_boolean"/>
                             <div class="content-group" name="amazon_connector" invisible="not module_sale_amazon"/>
                         </setting>
+                        <setting id="shopee_connector" help="Import Shopee orders and sync deliveries">
+                            <field name="module_sale_shopee" widget="upgrade_boolean"/>
+                            <div class="content-group" name="shopee_connector" invisible="not module_sale_shopee"/>
```

Comment:
is this div used anywhere? :thinking:

Reply:
Dunno what you are talking about :innocent: